### PR TITLE
Added geometry box type

### PR DESCRIPTION
**Pull request by mvrhov**

### CODE REVIEW

**Comment by chanmix51** [comment]
I do not know why the tests fail for Pg version prior to 9.2. We are going to drop the support for Pg 9.1 anyway so I must update the test accordingly. 
Your PR is nice, I will merge it on master manually.

File: sources/lib/Converter/Geometry/PgBox.php [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+     */
+    public function toPg($data, $type, Session $session)
+    {
+        if ($data === null) {
+            return sprintf("NULL::%s", $type);
+        }
+
+        $data = $this->checkData($data);
+
+        return sprintf(
```
Comment (sanpii) [inline sources/lib/Converter/Geometry/PgBox.php]: You can simply replace this code by:
```php
(string)$data;
```

Comment (mvrhov) [inline sources/lib/Converter/Geometry/PgBox.php]: almost

File: sources/lib/Converter/Geometry/PgBox.php [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+use PommProject\Foundation\Converter\ConverterInterface;
+use PommProject\Foundation\Session\Session;
+
+/**
+ * PgPoint
+ *
+ * Converter for PostgreSQL point type.
+ *
+ * @package Foundation
+ * @copyright 2014 Grégoire HUBERT
```
Comment (mvrhov) [inline sources/lib/Converter/Geometry/PgBox.php]: Fixed the year. Others were fixed before.

File: sources/lib/Converter/Type/Box.php [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+     * Create a box from a string description.
+     *
+     * @access public
+     * @param  string $description
+     */
+    public function __construct($description)
+    {
+        $description = trim($description, ' ()');
+
+        if (!preg_match('/([0-9e\-+\.]+), *([0-9e\-+\.]+) *\), *\( *([0-9e\-+\.]+), *([0-9e\-+\.]+)/', $description, $matches)) {
```
Comment (chanmix51) [inline sources/lib/Converter/Type/Box.php]: This line does not respect the coding conventions (PSR2), this is why it fails in Travis (php-cs inspection). You have to either split the call to `preg_match` or the store the regular expression in a separate variable.

**Pull request merged**